code/modules/mob/living/carbon/status_procs.dm

```diff
@@ -14,6 +14,7 @@
 	if(!IsParalyzed())
 		to_chat(src, "<span class='notice'>You're too exhausted to keep going...</span>")
 	stam_paralyzed = TRUE
+	Paralyze(50)
```

Comment:
since this touches things outside of adrenals it should be separate PR tbh.

Reply:
That would require that change be merged before this. I am not okay with making adrenals a sustained 100% anti-stun against sec weaponry, which we then have to fix afterwards.

Reply:
It's fine dude